python/mlx/nn/losses.py

```diff
@@ -70,8 +70,9 @@ def cross_entropy(
     targets_as_probs = targets.ndim == logits.ndim
 
     def _drop_dim(shape, axis):
+        shape = list(shape)
```

Comment:
Feel free to ignore but `return shape[:axis] + shape[axis+1:]`? Avoids the list/tuple conversion.

Reply:
Ah it's unfortunately a slightly more involved to get negative axes to work.

Reply:
Makes sense :-)